c_glib/gandiva-glib/function-registry.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#pragma once
+
+#include <arrow-glib/arrow-glib.h>
+
+G_BEGIN_DECLS
+
+#define GGANDIVA_TYPE_FUNCTION_REGISTRY (ggandiva_function_registry_get_type())
+G_DECLARE_DERIVABLE_TYPE(GGandivaFunctionRegistry,
+                         ggandiva_function_registry,
+                         GGANDIVA,
+                         FUNCTION_REGISTRY,
+                         GObject)
+
+struct _GGandivaFunctionRegistryClass
+{
+  GObjectClass parent_class;
+};
+
+GGandivaFunctionRegistry *ggandiva_function_registry_new();
+GList *ggandiva_function_registry_get_native_functions(GGandivaFunctionRegistry *registry);
```

Comment:
Could you use the same parameter name in `.h` and `.cpp`?
`registry` in `.h` and `function_registry` in `.cpp`.